Sources/FCM/FCMApnsConfig/FCMApnsApsObject.swift

```diff
@@ -95,7 +95,7 @@ public class FCMApnsApsObject: Codable {
                                  mutableContent: mutableContent))
     }
     
-    public convenience init(alert: FCMApnsAlert? = nil,
+    public init(alert: FCMApnsAlert? = nil,
                             badge: Int? = nil,
```

Comment:
Could you move `badge`, `sound`, `contentAvailable` a little bit left to look good as it was with `convenience` keyword before 🙂

Reply:
Indeed, my bad! fixed in ea45a4c